spinnaker_camera_driver/cmake/download_spinnaker

```diff
@@ -87,32 +183,16 @@ if not os.path.exists(os.path.join(os.getcwd(), "usr/lib/")):
     logging.info("Unpacking debs.")
     debs = glob.glob(os.path.join(os.getcwd(), folder_name, "libspinvideoencoder-*.deb"))
     debs += glob.glob(os.path.join(os.getcwd(), folder_name, "*spinnaker-*.deb"))
+    unpack_dir = os.path.join(os.getcwd(), 'flir_camera_driver', 'spinnaker_camera_driver')
     for deb in debs:
-        subprocess.call(['dpkg', '-x', deb, os.path.join(os.getcwd(), folder_name)])
-
-    # Fix for 1.13.0.31
-    os.remove(os.path.join(os.getcwd(), folder_name, "usr/lib/libspinnaker.so.1"))
-    os.symlink(shared_library, os.path.join(os.getcwd(), folder_name, "usr/lib/libSpinnaker.so.1"))
-
-    if not os.path.exists(os.path.join(os.getcwd(), "usr")):
-        os.mkdir(os.path.join(os.getcwd(), "usr"))
-
-    # For every folder/file, copy it into the /usr folder we just created.
-    for filename in os.listdir(os.path.join(os.getcwd(), folder_name, "usr")):
-        if not os.path.exists(os.path.join(os.getcwd(), "usr", filename)):
-            shutil.move(os.path.join(os.getcwd(), folder_name, "usr", filename), os.path.join(os.getcwd(), "usr", filename))
-
-    # now we do some thing for XML files; copy them next to the catkin binary
-    xml_path = os.path.join(os.getcwd(), "usr/lib/")
-    xml_files = os.listdir(xml_path)
-
-    try:
-        os.makedirs(destination_folder)
-    except:
-        logging.info("Error XML folder either exists or can't be created.")
-
-    for files in xml_files:
-        if files.endswith(".xml"):
-            origin_file = os.path.join(xml_path, files)
-            print('Copying ' + origin_file + ' to ' + destination_folder)
-            shutil.copy(origin_file, destination_folder)
+        logging.info('Extracting {0}'.format(deb))
+        subprocess.call(['dpkg', '--extract', deb, unpack_dir])
+
+    ## Fix for 1.13.0.31
+    #os.remove(os.path.join(unpack_dir, "usr/lib/libspinnaker.so.1"))
+    #os.symlink(shared_library, os.path.join(unpack_dir, "usr/lib/libSpinnaker.so.1"))
+
+    # finally, copy the shared libraries into the output folder
+    src_file = os.path.join(unpack_dir, ARCHS[arch]['shared_library'])
+    logging.info('Copying shared library to {0}'.format(destination_folder))
+    shutil.copyfile(src_file, os.path.join(destination_folder, 'libSpinnaker.so'))
```

Comment:
Do we only need one of the libraries? Do we not need the xml files?

Reply:
The original version didn't include the XML files either, so I assume they are not necessary.